app/assets/javascripts/expand.js

```diff
@@ -43,58 +43,52 @@ $(document).ready(function() {
         para5.setAttribute("class", "red-completion");
       }
 
-      para5.appendChild(complete_node);
-
-      var break_line = document.createElement("br");
+      // para5.appendChild(complete_node);
 
       var title_node = document.createTextNode(title);
-      title_header.appendChild(title_node);
-      title_header.appendChild(para5);
+      // title_header.appendChild(title_node);
+      // title_header.appendChild(para5);
 
       var datetime_format = d3.time.format("%m/%d/%Y %I:%M %p");
       var datetime_string = datetime_format(new Date(date_and_time));
       var datetime_node = document.createTextNode(datetime_string);
 
-      var element = document.getElementById("expand-event");
-
-      //Create an input type dynamically.
       var update_button = document.createElement("a");
-      var delete_button = document.createElement("a");
-
       var update_text = document.createTextNode("update event");
-      var delete_text = document.createTextNode("delete event");
 
       update_button.setAttribute('class','button tiny update-butt');
       update_button.setAttribute('href','/timelines/' + timeline_id + '/events/' + response.event.id + '/edit');
 
+      var delete_button = document.createElement("a");
+      var delete_text = document.createTextNode("delete event");
```

Comment:
Identifier 'delete_text' is not in camel case.<br>Mixed double and single quotes.